### DISCUSSION TITLE
Access rights custom fields

### DISCUSSION THREAD

**Discussion by HeinzWuert** [kimai/kimai#4149] [Plugins]
Hope someone can help:
With the plugin custom fieldsI have added some data fields in the user administration, which are to be filled in by the users themselves. Role_User permissions were granted for this.
Now the team leaders should have access to all user data, but not to the additionally created data fields.
Can this be restricted in any way?

**Comment by kevinpapst** [comment]
You could removed `preferences_other_profile` to hide all preferences from teamleads.
Which is not exactly what you want, but the only way I can think of right now.

**Comment by HeinzWuert** [comment]
Unfortunately it doesn't change anything

**Reply by kevinpapst** [reply]
Then permissions are wrongly configured. It should fully remove access to user preferences for Teamleads

**Comment by HeinzWuert** [comment]
So it is configured but It    does not help for seperating.
  

  

  
  

  
  
  
  
>
